std/typecons.d

```diff
@@ -5141,7 +5141,15 @@ therefore avoiding the overhead of $(D new). This facility is unsafe;
 it is the responsibility of the user to not escape a reference to the
 object outside the scope.
 
-Note: it's illegal to move a class reference even if you are sure there
+The class's destructor will be called when the result of `scoped()` is 
+itself destroyed.
+
+Class instances can be embedded in a `class` or `struct` as member 
```

Comment:
This sentence feels like a random fact. I see two points here: (a) With scoped class instance can be placed like struct instances and (b) the type `typeof(scoped!Class(args))` has to be used to use a scoped class as a member variable. The second point is actual information (I did not know that before). Maybe you can make this point more pronounced. In addition a one-sentence explanation would be nice.